src/sdJwt/disclosures.ts

```diff
@@ -6,6 +6,7 @@ export class Disclosure {
     private salt: string
     private key?: string
     private value: unknown
+    private _digest: string | undefined
```

Comment:
```suggestion
    private digest?: string
```

Was never a fan of the `_` prefix.

Reply:
This will clash with the method with the same name